sls/src/components/InfoSection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import 'typeface-roboto';
+import { Container } from '@material-ui/core';
+import { Button } from 'reactstrap';
+import Help from './Help.js';
+import Contact from './Contact.js';
+
+const styles = {
+  container: {
+    margin: '1vh auto',
+  },
+  buttonContainer: {
+    marginBottom: '15px',
+  },
+  button: {
+    fontSize: '1em',
+    margin: '0 10px',
+  },
+  collapseStyles: {
+    margin: '10px auto',
+  },
+  textBody: {
+    fontSize: '0.8em',
+  }
+}
+
+class InfoSection extends React.Component {
+  constructor(props) {
```

Comment:
let's try to get rid of this constructor and `bind` keyword.

To define state, use:
```
state= {
 ...state
}
```

You dont need to be binding class members